scanner/indexer/service.go

```diff
@@ -1,12 +1,44 @@
 package indexer
 
-// TODO: Make this into a gRPC service.
-type Service struct {
+import (
+	"context"
+
+	"github.com/gogo/protobuf/types"
+	"github.com/grpc-ecosystem/grpc-gateway/runtime"
+	"github.com/stackrox/rox/generated/internalapi/scanner/v4"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+type indexerService struct {
+	v4.UnimplementedIndexerServer
 	indexer *Indexer
 }
 
-func NewService(indexer *Indexer) (*Service, error) {
-	return &Service{
+func NewIndexerService(indexer *Indexer) (*indexerService, error) {
+	return &indexerService{
 		indexer: indexer,
 	}, nil
 }
+
+func (s *indexerService) CreateIndexReport(ctx context.Context, req *v4.CreateIndexReportRequest) (*v4.IndexReport, error) {
+	return nil, status.Errorf(codes.Unimplemented, "method CreateIndexReport not implemented")
+}
```

Comment:
sorry missed this last round. Can you add new line between these functions, too?

Reply:
That's OK, I will follow up in the child PR.